docs/reference/ml/apis/evaluateresources.asciidoc

```diff
@@ -0,0 +1,63 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ml-evaluate-dfanalytics-resources]]
+=== {dfanalytics-cap} evaluation resources
+
+Evaluation configuration objects relate to the <<evaluate-dfanalytics>>.
+
+[discrete]
+[[ml-evaluate-dfanalytics-properties]]
+==== {api-definitions-title}
+
+`evaluation`::
+  (object) Defines the type of evaluation you want to perform. The value of this 
+  object can be different depending on the type of evaluation you want to 
+  perform. For example, it can contain <<binary-sc-resources>>.
+
+[[binary-sc-resources]]
+==== Binary soft classification configuration objects
+
+Binary soft classification evaluates the results of an analysis which outputs 
+the probability that each {dataframe} row belongs to a certain class. For 
+example, in the context of outlier detection, the analysis outputs the 
+probability whether each row is an outlier.
+
+[discrete]
+[[binary-sc-resources-properties]]
+===== {api-definitions-title}
+
+`actual_field` (Required)::
```

Comment:
Typically this information about which fields are required or optional appears in the API reference page. It's unusual to see it here.

Reply:
I removed the notes in fc3ad12.